libs/api/app/data-access/src/lib/entity/app-transaction-error.entity.ts

```diff
@@ -0,0 +1,15 @@
+import { Field, Int, ObjectType } from '@nestjs/graphql'
+import { ApiProperty } from '@nestjs/swagger'
+import { AppTransactionErrorType } from '@prisma/client'
```

Comment:
This one should come from `./app-transaction-error-type.enum` as it has the GraphQL metadata.